### DISCUSSION TITLE
Minimal Mistakes remote theme starter yields different _config.yml

### DISCUSSION THREAD

**Discussion by ghost** [mmistakes/minimal-mistakes#3173] [Q&A] [answered]
Hi,

Why does Minimal Mistakes remote theme starter yield a different _config.yml file than minimal-mistakes/_config.yml?

**Comment by mmistakes** [comment, marked as answer]
The starter is a simplified `_config.yml` with the "bare bones" of settings. While the one in these repo is more exhaustive with pretty much all options added for demo purposes.

As both are meant to be changed by you they don't need to be the same.

**Reply by ghost** [reply]
Can you please name the other differences? I hadn't had expected the starter to be a minimal working example (MWE). I would like to have to 'full' experience.

**Reply by iBug** [reply]
If you want a "full" experience, just copy `_config.yml` from the theme repo and customize as needed. No need to name every difference (it'd be boring).

**Reply by ghost** [reply]
I've just copied `_config.yml` but it breaks the page:

<img width="1062" alt="Screen Shot 2021-09-30 at 8 23 20 AM" src="https://user-images.githubusercontent.com/31712384/135398648-03e0d206-28ca-4acd-9cea-9a909d1e38c2.png">

What do I need to do additionally?

**Reply by iBug** [reply]
You certainly should have done this

> copy `_config.yml` from the theme repo and **customize as needed**

The "boilerplace" config is there to show you what options are available, not "directly available for use".

**Reply by ghost** [reply]
That's why it would be nice to have all options in the starter, even if commented out. Now I have no idea what is breaking it and have no minimal working example.

> they don't need to be the same.
but could be as I understand
